library/Fission/Internal/Orphanage/RegisterDid.hs

```diff
@@ -0,0 +1,18 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Fission.Internal.Orphanage.RegisterDid () where
+
+
+import Fission.Prelude
+
+import Servant
+import Servant.Swagger
+import Data.Swagger
+
+import Fission.Web.Auth.Types
+
+-- @@TODO: add an accurate security definition
+instance HasSwagger api => HasSwagger (RegisterDid :> api) where
```

Comment:
This probably doesn't need to be an orphan instance 🤔 We have `InstanceSigs` turned on in #252 that should probably help with this. Ping me for more info.